Source/FocusTuner/FocusTuner.cs

```diff
@@ -0,0 +1,138 @@
+            public int CalibrationID;
+            /// <summary>Specifies the S-parameters of the adapter section between the DUT and the tuner at the primary frequency and up to four secondary frequencies.</summary>
+            public SParameters[] DUTtoTunerSParameters;
+            /// <summary>Specifies the reflection coefficient of the termination seen by the tuner at load side for the primary frequency and up to four secondary frequencies.</summary>
+            public Complex[] LoadTermination;
+
+            /// <summary>Returns the struct with default values set.</summary>
+            /// <returns>The struct with default values set.</returns>
+            public static CommonConfiguration GetDefault()
+            {
+                var commonConfiguration = new CommonConfiguration
+                {
+                    TunerMode = TunerMode.Load,
+                    CalibrationID = 1,
+                    DUTtoTunerSParameters = new SParameters[1],
+                    LoadTermination = new Complex[1]
+                };
+                commonConfiguration.DUTtoTunerSParameters[0] = SParameters.GetDefault();
+                commonConfiguration.LoadTermination[0] = new Complex { Real = 0, Imaginary = 0 };
+
+                return commonConfiguration;
+            }
+        }
+        #endregion
+
+        #region Configuration
+        /// <summary>Reinitializes all the axes of the tuner to its home position before terminating the software connection to the instrument.</summary>
+        /// <param name="iTuner">Specifies a reference to the instrument.</param>
+        public static void CloseTuner(FocusITunerBroker iTuner)
+        {
+            iTuner.InitializeTunerAllAxises();
+            iTuner.Close();
+        }
+
+        /// <summary>Configures common settings for load pull using the Focus tuner.</summary>
+        /// <param name="iTuner">Specifies a reference to the instrument.</param>
+        /// <param name="commonConfiguration">Specifies the settings for load pull using the Focus tuner.</param>
+        /// <returns>The primary frequency in GHz, of the active tuner calibration data set.</returns>
+        public static double ConfigureCommon(FocusITunerBroker iTuner, CommonConfiguration commonConfiguration)
+        {
+            List<Complex[]> sParameters = new List<Complex[]>();
+            foreach (var element in commonConfiguration.DUTtoTunerSParameters)
+            {
+                Complex[] sParameter = new Complex[4];
+                //The order in the Focus driver is S11, S12, S21 and S22.
+                sParameter[0] = element.S11;
+                sParameter[1] = element.S12;
+                sParameter[2] = element.S21;
+                sParameter[3] = element.S22;
+                sParameters.Add(sParameter);
+            }
```

Comment:
```suggestion
            Complex[][] sParameters = new Complex[commonConfiguration.DUTtoTunerSParameters.Length][];
            for (int i = 0; i < sParameters.Length; i++)
            {
                SParameters element = commonConfiguration.DUTtoTunerSParameters[i];
                sParameters[i] = new Complex[] { element.S11, element.S12, element.S21, element.S22 };
            }
```
This comment is merely a suggestion. You're code works and is fine but i believe the above suggestion is more concise and easier to read. Furthermore it removes the dependency on Collections.Common. Up to you if you want to use it.

Reply:
Accepted. Thanks for the explanation.